etc/systemd/system/restic-backup@.service

```diff
@@ -8,5 +8,6 @@ Type=simple
 Nice=10
 # $HOME or $XDG_CACHE_HOME must be set for restic to find /root/.cache/restic/
 Environment="HOME=/root"
+# The random sleep (in seconds) is in the case of multiple backup profiles. Many restic instances started at the same time could case high load or network bandwith usage.
 # `systemd-cat` allows showing the restic output to the systemd journal
-ExecStart=bash -c 'source /etc/restic/%I.env && /usr/local/sbin/restic_backup.sh | systemd-cat'
+ExecStart=bash -c 'sleep ${RANDOM:0:300} && source /etc/restic/%I.env && /usr/local/sbin/restic_backup.sh | systemd-cat'
```

Comment:
I didn't know the existence of the built-in `$RANDOM`. But the bounds: `:0:300` does not seem to work in this way.

Using `${VAR:x:y}` I think it is taken as a string, where `x` is the starting position, and optionally `y` the length. I run it and I got this:

```console
❯ echo ${RANDOM:0:300}
22519

❯ echo ${RANDOM:0:1}
7
```

Reply:
This seems to work:
`echo $((RANDOM % 300))` 
or
` echo $RANDOM % 10 + 1 | bc`

Reply:
Thanks for checking!